src/components/AddUserForm/AddUserForm.js

```diff
@@ -0,0 +1,147 @@
+import React, {Component, createRef} from 'react';
+import uniqId from 'uniqid';
+import {Button} from '../Button/Button';
+import './AddUserForm.scss';
+import ImgWarning from './../../assets/warning2.png'
+
+const CN = 'user-form';
+
+class AddUserForm extends Component {
+    constructor(props) {
+        super(props);
+        // this.userNameRef = createRef(); // аналогично записи в строке 19
+        // this.userLastNameRef = createRef();
+        this.state = {
+            currentError: '',
+            errorUserName: ' ',
+            errorUserLastName: ' '
+        };
+    }
+
+    // todo: добавить state, в него добавить пропертю, которая будет отображать есть ли ошибка или нет.
+    //  допустим это будет строка currentError
+    //  по умолчанию она будет пустой
+    //  если произошла ошибка - в эту пропертю запишется строка, какая именно ошибка произошла
+
+    userNameRef = createRef();
+    userLastNameRef = createRef();
+
+    onInputChange = (e) => {
+        console.log(e.target.value);
+        // todo: если в стейте была записана какая-то ошибка - зачищаем ее при изменении инпута
+        //  поскольку в ситуации если пользователь ничего - не ввел - увидел ошибку - понял, что не так - начал вводить данные
+        //  было бы хорошо не пугать его висящим сообщением, что все таки что-то не так
+        this.setState({
+            currentError: '',
+            // errorUserName: '',
+            // errorUserLastName: ''
+            });
+        if (this.userNameRef.current.value !=='') {this.setState({errorUserName: ''})}
+        if (this.userLastNameRef.current.value !=='') {this.setState({errorUserLastName: ''})}
```

Comment:
Also you can add condition to check if there is error in current moment
It would help you to avoid  extra rerenders

like this 
````
  this.state.currentError && this.setState({
            currentError: '',
            // errorUserName: '',
            // errorUserLastName: ''
            });
...
 if (this.userNameRef.current.value !=='' && this.state.errorUserName) {this.setState({errorUserName: ''})}

````